src/shim/hwctx.cpp

```diff
@@ -244,14 +244,30 @@ delete_ctx_on_device()
   fini_log_buf();
 }
 
+void
+hw_ctx::
+set_metadata()
+{
+  m_metadata.magic_no = 0xBAADBEEF;
+  m_metadata.major = 1;
+  m_metadata.minor = 4;
```

Comment:
this doesn't have to be the same as mailbox. let's set it to 0.1
major 0
minor 1